net/address/address.go

```diff
@@ -23,6 +32,18 @@ func (r Range) Size() Count                { return Length(r.End, r.Start) }
 func (r Range) String() string             { return fmt.Sprintf("%s-%s", r.Start, r.End-1) }
 func (r Range) Overlaps(or Range) bool     { return !(r.Start >= or.End || r.End <= or.Start) }
 func (r Range) Contains(addr Address) bool { return addr >= r.Start && addr < r.End }
+func (r Range) Equals(or Range) bool       { return r.Start == or.Start && r.End == or.End }
+
+// IsCIDR checks whether the range is CIDR.
+func (r Range) IsCIDR() bool {
```

Comment:
This is only called in an `assert` and it seems to me that if you change `ipam.Config.Universe` to a CIDR then you don't ever need to do this check.